test/spec/modules/nextrollIdSystem_spec.js

```diff
@@ -0,0 +1,56 @@
+import { nextrollIdSubmodule, storage } from 'modules/nextrollIdSystem.js';
+
+const LS_VALUE = `{
+  "AdID":{"id":"adid","key":"AdID"},
+  "AdID:1002": {"id":"adid","key":"AdID:1002","value":"id_value"}}`;
+
+describe('NextrollId module', function () {
+  let sandbox = sinon.sandbox.create();
+  let hasLocalStorageStub;
+  let getLocalStorageStub;
+
+  beforeEach(function() {
+    hasLocalStorageStub = sandbox.stub(storage, 'hasLocalStorage');
+    getLocalStorageStub = sandbox.stub(storage, 'getDataFromLocalStorage');
+  });
+
+  afterEach(function () {
+    sandbox.restore();
+  })
+
+  const testCases = [
+    {
+      expect: {
+        id: {nextroll: 'id_value'},
```

Comment:
```suggestion
        nextrollId: 'id_value',
```

Reply:
See https://github.com/prebid/Prebid.js/pull/6396#discussion_r604393088 . This test case is correct, it's indeed confusing, but the value is processed differently when it's not the result of the callback.

I've made sure to double check in a testpage.

Reply:
you're right, all good